File: Shopping-Interface/src/main/java/projectInterface/TypeTemplateService.java
package projectInterface;

import entity.PageResult;
import pojo.typeTemplate.TbTypeTemplate; /*********************************************************
 ** Package: projectInterface<br>
 ** Description： TODO <br>
 ** Date: Created in 2018/10/18 17:29<br>
 ** Copyright: Copyright (c) 2018<br>
 *********************************************************/
public interface TypeTemplateService
{
	/**
	 * @Description -模板管理的条件分页查询
	 * @Date        -2018/10/18  17:45
	 * @para        -
	 * */
	PageResult findPage(int pageNum, int pageSize, TbTypeTemplate tbTypeTemplate);
}

File: Shopping-Web-Manager/src/main/webapp/js/service/typeTemplateService.js

app.service("typeTemplateService",function ($http) {

    //分页查询数据
    this.findPage = function (pageNum, pageSize,searchContent)
    {
        return $http.post('../typeTemplate/getTypeTemplateMessage.action?pageNum=' + pageNum + '&pageSize=' + pageSize, searchContent);
    }
});